src/ResourceManager/Sql/Commands.Sql/help/Get-AzureRmSqlDatabaseLongTermRetentionBackup.md

```diff
@@ -0,0 +1,332 @@
+ServerName           : server01
+ServerCreateTime     : 2/29/2018 12:12:19 AM
+```
+
+This command gets all long term retention backups for the northeurope location.
+
+### Example 2: Get a specific long term retention backup
+```powershell
+PS C:\> Get-AzureRmSqlDatabaseLongTermRetentionBackup -LocationName northeurope -ServerName server01 -DatabaseName database01 -BackupName "601061b7-d10b-46e0-bf77-a2bfb16a6add;131655666550000000"
+
+
+BackupExpirationTime : 3/22/2018 5:50:55 AM
+BackupName           : 601061b7-d10b-46e0-bf77-a2bfb16a6add;131655666550000000
+BackupTime           : 3/15/2018 5:50:55 AM
+DatabaseName         : database01
+DatabaseDeletionTime :
+LocationName         : northeurope
+ResourceId           : /subscriptions/371edd6d-9630-4558-a7bd-ee139498e6a1/providers/Microsoft.Sql/locations/northeurope/longTermRetentionServers/server01/longTermRetentionDatabases/database01/longTermRetentionBackups/601061b7-d10b-46e0-bf77-a2bfb16a6add;131655666550000000
+ServerName           : server01
+ServerCreateTime     : 2/29/2018 12:12:19 AM
+```
+
+This command gets the backup with name 601061b7-d10b-46e0-bf77-a2bfb16a6add;131655666550000000
+
+### Example 3: Get all long term retention backups for a database
+```powershell
+PS C:\> Get-AzureRmSqlDatabase -ResourceGroupName resourcegroup01 -ServerName server01 -DatabaseName database01 | Get-AzureRmSqlDatabaseLongTermRetentionBackup
+
+
+BackupExpirationTime : 3/22/2018 5:50:55 AM
+BackupName           : 601061b7-d10b-46e0-bf77-a2bfb16a6add;131655666550000000
+BackupTime           : 3/15/2018 5:50:55 AM
+DatabaseName         : database01
+DatabaseDeletionTime :
+LocationName         : northeurope
+ResourceId           : /subscriptions/371edd6d-9630-4558-a7bd-ee139498e6a1/providers/Microsoft.Sql/locations/northeurope/longTermRetentionServers/server01/longTermRetentionDatabases/database01/longTermRetentionBackups/601061b7-d10b-46e0-bf77-a2bfb16a6add;131655666550000000
+ServerName           : server01
+ServerCreateTime     : 2/29/2018 12:12:19 AM
+```
+
+This command gets all long term retention backups for database01
+
+## PARAMETERS
+
+### -BackupName
+The name of the backup.
+
+```yaml
+Type: String
+Parameter Sets: BackupName, GetBackupByResourceId, GetBackupByInputObject
+Aliases:
+
+Required: True
+Position: 3
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -DatabaseName
+The name of the Azure SQL Database the backup is from.
+
+```yaml
+Type: String
+Parameter Sets: DatabaseName, BackupName
+Aliases:
+
+Required: True
+Position: 2
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -DatabaseState
+The state of the database whose backups you want to find, Alive, Deleted, or All.
```

Comment:
"Alive" or "Live"?

Reply:
Alive. That's what we have all the way down the stack and in the .NET SDK.